qa/tasks/cephfs/mount.py

```diff
@@ -106,13 +127,13 @@ def _setup_brx_and_nat(self):
         if not brx:
             log.info("Setuping the 'ceph-brx' with {0}/{1}".format(ip, mask))
 
-            args = ["sudo", "bash", "-c", "ip link add name ceph-brx type bridge"]
-            self.client_remote.run(args=args, timeout=(5*60), omit_sudo=False)
-            args = ["sudo", "bash", "-c", "ip link set ceph-brx up"]
-            self.client_remote.run(args=args, timeout=(5*60), omit_sudo=False)
-            args = ["sudo", "bash", "-c",
-                    "ip addr add {0}/{1} brd {2} dev ceph-brx".format(ip, mask, brd)]
-            self.client_remote.run(args=args, timeout=(5*60), omit_sudo=False)
+            self.run_shell_payload(f"""
+                set -e
+                sudo ip link add name ceph-brx type bridge
+                sudo ip addr flush dev ceph-brx
+                sudo ip link set ceph-brx up
+                sudo ip addr add {ip}/{mask} brd {brd} dev ceph-brx
+            """, cwd=os.getcwd(), timeout=(5*60), omit_sudo=False)
```

Comment:
why `cwd=os.getcwd()`? This path does not exist on the mount machine:

````
2020-07-16T20:09:51.829 INFO:teuthology.orchestra.run:Running command with timeout 300
2020-07-16T20:09:51.829 INFO:teuthology.orchestra.run.smithi029:> (cd /home/teuthworker && exec sudo bash -c '
2020-07-16T20:09:51.830 INFO:teuthology.orchestra.run.smithi029:>                 set -e
2020-07-16T20:09:51.830 INFO:teuthology.orchestra.run.smithi029:>                 sudo ip link add name ceph-brx type bridge
2020-07-16T20:09:51.831 INFO:teuthology.orchestra.run.smithi029:>                 sudo ip addr flush dev ceph-brx
2020-07-16T20:09:51.832 INFO:teuthology.orchestra.run.smithi029:>                 sudo ip link set ceph-brx up
2020-07-16T20:09:51.834 INFO:teuthology.orchestra.run.smithi029:>                 sudo ip addr add 192.168.255.254/16 brd 192.168.255.255 dev ceph-brx
2020-07-16T20:09:51.834 INFO:teuthology.orchestra.run.smithi029:>             ')
2020-07-16T20:09:51.881 INFO:teuthology.orchestra.run.smithi029.stderr:bash: line 0: cd: /home/teuthworker: No such file or directory
````

https://pulpito.ceph.com/pdonnell-2020-07-16_18:53:18-fs-wip-pdonnell-testing-20200716.140830-distro-basic-smithi/5231339/

Reply:
Yeah, the os.getcwd() is the local work directory, it may not exist in the remote.